decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -174,8 +174,8 @@ def self.with_evaluation_assigned_to(user, space)
                           A: :title,
                           datetime: :published_at
                         },
-                        index_on_create: ->(proposal) { proposal.official? },
-                        index_on_update: ->(proposal) { proposal.visible? })
+                        index_on_create: ->(proposal) { proposal.official? && proposal.component&.published? },
```

Comment:
I am just curious why only the "offical proposals" are getting indexed :) 
```suggestion
                        index_on_create: ->(proposal) { proposal.visible? && proposal.component&.published? },
```